ga4gh/server/exceptions.py

```diff
@@ -102,7 +102,11 @@ class BadRequestException(RuntimeException):
     A request that we don't like was sent to the server.
     """
     httpStatus = 400
-    message = "Bad request"
+
+    def __init__(self, msg=None):
+        self.message = "Bad request"
+        if msg is not None:
+            self.message += ": " + msg
```

Comment:
Writing detailed warning message is always good practice and should be encouraged. This however provides a fallback for scenarios when no error messages are written, it will only return a plan 'Bad request' message.